rules/integrations/aws/aws_bedrock_high_resource_consumption_detection.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2024/05/04"
+maturity = "production"
```

Comment:
No integration tag?

Reply:
Actually, lets leave them off of ESQL rules for now since we cannot validate anyway